.vscode/launch.json

```diff
@@ -211,6 +211,13 @@
 				"Launch azuredatastudio",
 				"Attach to Extension Host"
 			]
-		}
+		},
+		{
+			"name": "Debug Unit Tests",
+			"configurations": [
+				"Attach to VS Code",
+				"Run Unit Tests"
+			]
+		},
```

Comment:
we already have this definition above, we should unify them.

Reply:
I think we just want to remove it right?